open_academy/models/inherited_partner.py

```diff
@@ -0,0 +1,8 @@
+from odoo import models, fields
+
+
+class InheritedPartner(models.Model):
+	_inherit = 'res.partner'
+
+	is_instructor = fields.Boolean('Is an instructor')
```

Comment:
I don't know if you are trying to add some help in the field, if it is the case change to

```suggestion
	is_instructor = fields.Boolean(help='This partner is a session instructor?')
```